xml/ha_storage_basics.xml

```diff
@@ -73,43 +73,47 @@
    <para>
     If you have a setup that includes both &ocfs; and &clvm;, configuring
     <emphasis>one</emphasis> DLM resource for both &ocfs; and &clvm; is enough.
+    In this case, configure DLM using <xref linkend="pro-dlm-resources"/>.
+   </para>
+   <para>
+    However, if you need to keep the resources that use DLM independent from one
+    another (such as multiple &ocfs; mount points), use separate colocation and
```

Comment:
```suggestion
    However, if you need to keep the resources that use DLM
    independently of one another (such as multiple &ocfs;
    mount points), use separate colocation and
```

Reply:
I don't think this change makes sense, unless the word "keep" were being used to mean something more like "manage", which it isn't. 

Some alternative suggestions:

- "However, if you need to keep the resources that use DLM separate from one another"
- "However, if the resources that use DLM need to be independent"
- "However, if you need to avoid dependecies between the resources that use DLM"

What do you think?

Reply:
Oh, I see the point of the phrase now! Let's keep it as is then. Thank you :-)

Reply:
Okay, thanks! :)